src/widgets/current-refined-values/current-refined-values.js

```diff
@@ -233,7 +233,7 @@ function clearRefinementFromState(state, refinement) {
   case 'exclude':
     return state.removeExcludeRefinement(refinement.attributeName, refinement.name);
   case 'numeric':
-    return state.removeNumericRefinement(refinement.attributeName, refinement.operator, refinement.name);
```

Comment:
To be consistent, I think `name` should be a string value.I would have used a numeric value if the attribute name was `value`.